### ISSUE TITLE
Analytical engine rework

### ISSUE THREAD

**Issue by Tat011** [captaincobbs/ESG-Mod#1491]
+3 dust per anomaly discovered is way too strong to be RNG so here's some fixes we can do:
-Put it on the tech tree.
-Nerf it to +1
-The fun one which is to nerf it to +2, add industry, food and science versions that are also found on curios and you can only build one of the four.
-A different version of the above where it's +1 and you can build as many of them as you want but only one of the four per individual system.

**Comment by captaincobbs** [comment]
You could cap it instead, up to +100, I don't really have an opinion on this though since it's more of a balancing issue, see what Waky says about it

**Comment by wakystuf** [comment]
Good idea on the cap; 100 seems reasonable to me. I'm also fine putting it on the tech tree, though that's less critical if there is a cap.
